src/pages/blogs/index.page.tsx

```diff
@@ -45,9 +45,9 @@ const Index = ({ blogs, populars, categories, pageCount }: Props) => {
       <Page>
         <div className={styles.banner}>
           <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerRight} alt="cell" className={styles.right} />
+          {/* <Image src={blogBannerLeft} alt="cell" className={styles.left} /> */}
           <Image src={blogBannerRight} alt="cell" className={styles.right} />
+          {/* <Image src={blogBannerRight} alt="cell" className={styles.right} /> */}
```

Comment:
need to be removed

Reply:
```suggestion
```

Reply:
Updated by https://github.com/Magickbase/nervos-official-website/pull/36/commits/e8e5e0c721e5a95c8a47431c3521b961b308b633